src/dbus_helper/mod.rs

```diff
@@ -25,3 +30,77 @@ macro_rules! dbus_connect {
             .unwrap_or_else(|e| panic!("dbus_connect error: {}", e))
     }}
 }
+
+#[inline]
```

Comment:
Why have you used `inline` for most functions? The rust reference says "The compiler automatically inlines functions based on internal heuristics. Incorrectly inlining functions can actually make the program slower, so it should be used with care"